core/src/main/java/io/questdb/cairo/wal/WalWriterEvents.java

```diff
@@ -262,11 +264,17 @@ void of(ObjList<CharSequenceIntHashMap> txnSymbolMaps, AtomicIntList initialSymb
         this.symbolMapNullFlags = symbolMapNullFlags;
     }
 
-    void openEventFile(Path path, int pathLen, boolean truncate) {
+    void openEventFile(Path path, int pathLen, boolean truncate, boolean systemTable) {
         if (eventMem.getFd() > -1) {
             close(truncate, Vm.TRUNCATE_TO_POINTER);
         }
-        openSmallFile(ff, path, pathLen, eventMem, EVENT_FILE_NAME, MemoryTag.MMAP_TABLE_WAL_WRITER);
+        eventMem.of(
+                ff,
+                path.trimTo(pathLen).concat(EVENT_FILE_NAME).$(),
+                systemTable ? configuration.getSystemWalEventAppendPageSize() : configuration.getWalEventAppendPageSize(),
+                MemoryTag.MMAP_TABLE_WAL_WRITER,
+                CairoConfiguration.O_NONE
```

Comment:
```suggestion
                Files.POSIX_MADV_RANDOM
```

Reply:
those opts are flags on "open" https://man7.org/linux/man-pages/man2/open.2.html

they are exposed for O_DIRECT mainly.

Reply:
fixed in https://github.com/questdb/questdb/pull/4221/commits/acdf66cdbc0a17413c376abd4135f22aa0d45089